Data/Comtrade/get_comtrade.py

```diff
@@ -1,21 +1,19 @@
-# Download and format data from Comtrade API,
-# keeping a log of successful downloads and errors.
+# Download and format data from Comtrade API
 
 import pandas as pd
 import math
 import os
 import json
 from urllib.request import urlopen
 import time
-import csv
-import numpy as np
 
-print(os.getcwd())
-os.chdir(".")  # set your current directory
+# import csv
+import numpy as np
 
-# create a directory to save downloaded data
-if not os.path.exists("csv"):
-    os.makedirs("csv")
+model_name = "slf_model"

```

Comment:
Recommend we move this to below the function definition and change it to read from an .env file. For example:

```suggestion
model_name = "slf_model"
from dotenv import load_dotenv
project_path = 'H:/Shared drives/APHIS  Projects/Pandemic # ex location in Google Drive to store .env file
load_dotenv(os.path.join(project_path, '.env'))

# Root project data folder
data_path = os.getenv('DATA_PATH')

# Path to formatted model inputs
input_dir = data_path + "slf_model/inputs/"
```

Reply:
I added these lines, but did not test with env file.
Should we get the model name from the env file too so its more flexible?